core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -341,6 +341,13 @@ trait AttributeComponent {
       }
     }
 
+    def deleteAttributeColumn(workspaceNamespace: String, workspaceName: String, entityType: String, entityNamespace: String, attributeName: String) = {
+     sql"""delete ea from ENTITY_ATTRIBUTE ea
+               join ENTITY e on e.id = ea.owner_id
+               join WORKSPACE w on w.id = e.workspace_id
+           where w.namespace = ${workspaceNamespace} AND w.name = ${workspaceName} AND e.entity_type = ${entityType} AND ea.namespace = ${entityNamespace} AND ea.name = ${attributeName}""".as[Int]
+    }
```

Comment:
could you have this accept a workspace UUID instead of workspaceNamespace + workspaceName? In that case you could omit the join to the `WORKSPACE` table and the `where w.namespace = ${workspaceNamespace} AND w.name = ${workspaceName}`, replacing it with `where e.workspace_id = $workspaceId`